CIP-Fair_Stakepool_Rewards/README.md

```diff
@@ -0,0 +1,176 @@
+
+To ensure a more effective pledge benefit and incentivize increasing pledge over time, this proposal increases the current value of k from 500 to 1000. This allows a fully pledged pledge benefit to be closer for all SPOs and will force multipools to split pledge and reduce pledge benefit if they wish to continue operating with the same leverage. Additionally, a change in the value of k will give many stagnant delegators an incentive to reconsider their delegations giving smaller stakepools an opportunity at increasing delegation.
+
+Finally, to reduce the large rewards disparity between private pools and delegators, this proposal reduces the setting of a0 from 0.3 to 0.2. The current setting of a0 results in approximately 30% greater rewards for fully pledged private pools. This proposal reduces this disparity to 20% to create a fairer rewards distribution. The result is an overall increase in rewards for delegators as most public pools operate with low pledges relative to the saturation point. Given that delegators make up majority of users, this reduction in a0 will make Cardano a much more competitive staking investment in contrast to other blockchains.
+
+These proposed changes to Cardano’s reward sharing scheme are aimed at ensuring fairness, increasing decentralization, and creating a more egalitarian staking ecosystem.
+
+## Test Cases
+
+Stakepool viability and competitive points can give some insight into the fairness of the reward scheme. These points are essentially start-up costs required to run viable and competitive stakepools. These points are very high and out of reach for many SPOs with the current scheme. This proposal effectively minimizes these points.
+
+Current stakepool viability point: ~625,000 ADA
+
+Current stakepool competitive point: ~19,000,000 ADA
+
+Proposal stakepool viability point: 1 ADA
+
+Proposal stakepool competitive point: 1 ADA
+
+See FairStakepoolRewards.xlxs to compare stakepool ROS between the current and proposed scheme.
+
+# Backwards Compatibility
+
+This proposal includes parameter changes, one parameter removal, and a change to the rewards calculation. Because of the parameter removal and changes to the rewards calculation, a hardfork will be necessary for implementation.
+
+# Path to Active
+
+This proposal must be approved by both IOG as well as the community before implementation. Community consensus should be a high priority for proposals that modify staking rewards.
+
+Implementation can be staged to reduce shock to the network:
```

Comment:
I'd encourage to clearly define an implementor in this plan. In particular, if it is to be people working at IOG, they would need to sign off on this plan.